packages/@webex/internal-plugin-calendar/src/calendar.js

```diff
@@ -245,15 +245,26 @@ const Calendar = WebexPlugin.extend({
   },
 
   /**
-   * Retrieves an array of meeting participants for the meeting id
-   * @param {String} id
+   * Retrieves an array of meeting participants for the meeting participantsUrl
+   * @param {String} participantsUrl
    * @returns {Promise} Resolves with an object of meeting participants
    */
-  getParticipants(id) {
+  getParticipants(participantsUrl) {
     return this.request({
       method: 'GET',
-      service: 'calendar',
-      resource: `calendarEvents/${btoa(id)}/participants`,
+      uri: participantsUrl,
+    });
+  },
+
+  /**
+   * Retrieves a collection of meetings based on the request parameters
```

Comment:
this comment doesn't seem to match what the function is doing

Reply:
copy from getNotes function. Maybe i changed to get meeting notes using notesUrl from meeting object.